src/main/java/com/iexec/blockchain/security/WebSecurityConfigurationAdapter.java

```diff
@@ -20,14 +20,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .csrf().disable()
                 .cors().and()
                 .authorizeRequests()
-                .mvcMatchers("/v2/api-docs", // <--- START Anonymous swagger access
-                        "/configuration/ui",
-                        "/swagger-resources/**",
-                        "/configuration/security",
-                        "/swagger-ui.html",
-                        "/swagger-ui/**",
-                        "/webjars/**").anonymous() // <--- END Anonymous swagger access
-                .antMatchers("/actuator/health", "/config/chain").permitAll()
+                .mvcMatchers("/swagger-ui.html", "/swagger-ui/**", "/v3/api-docs/**").anonymous() // Anonymous swagger access
```

Comment:
Minor, I think non-oneline is easier auditable:
```
"/bla.html",
"/blu.html"
```

Reply:
see 04cbb95